Rakefile

```diff
@@ -102,3 +103,21 @@ task :deploy do
   end
 end
 
+desc 'Test build'
+task :test do
+  sh "bundle exec jekyll doctor"
+  sh "bundle exec jekyll build --config _config.yml,_config_test.yml"
+  HTMLProofer.check_directory('./_site', {
+    :url_ignore => [
+      /relishapp.com/, # blacklisted?
+      /nike.com/, # blacklisted?
+      /apiblueprint.org/, # SSL issue
```

Comment:
<img width="682" alt="snimek obrazovky 2016-06-24 v 14 22 47" src="https://cloud.githubusercontent.com/assets/1788727/16337620/285be198-3a17-11e6-8521-1a49e0ac3984.png">
I'm getting this locally - using http://httpie.org - works with plain curl

And see this build error: https://circleci.com/gh/apiaryio/apiaryio.github.com/197 - sometimes I get error code 0, sometimes 301 SSL error